src/extension/features/toolkit-reports/pages/income-vs-expense/components/monthly-transaction-totals-table/components/collapsable-row/component.tsx

```diff
@@ -68,6 +78,6 @@ export class CollapsibleRow extends React.Component {
   }
 
   _toggleCollapse = () => {
-    this.props.onToggleCollapse(this.props.source?.entityId ?? this.props.source?.get('entityId'));
+    this.props.onToggleCollapse(this.props.source?.entityId!);
```

Comment:
The `!` doesn't look like it is serving a purpose since it seems to have been happy accepting `undefined` before.